cdap-app-fabric/src/main/java/co/cask/cdap/metadata/DefaultEntityExistenceVerifier.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright © 2015-2016 Cask Data, Inc.
```

Comment:
hmmm, this got renamed from ``EntityValidator``. not sure why github thinks its a new file